### PR TITLE
PT-2232 Release Notes 3.5.4

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
modified:   Changelog
	modified:   Makefile.PL
	modified:   config/sphinx-build/conf.py
	modified:   docs/release_notes.rst

- [ ] The contributed code is licensed under GPL v2.0
- [ ] Contributor Licence Agreement (CLA) is signed
- [ ] util/update-modules has been ran
- [x] Documention updated
- [ ] Test suite update

### CODE REVIEW

**Review by patrickbirch — APPROVED** [review]
LGTM

File: Changelog [hunk 1]
```diff
@@ -1,5 +1,13 @@
 Changelog for Percona Toolkit
 
+v3.5.4 released 2023-06-30
+
+* Improvement PT-2151: Fix tests for pt-query-digest
```
Comment (svetasmirnova) [inline Changelog:5]:
```suggestion
* Improvement PT-2151: Fix tests for pt-query-digest
* Improvement PR-614:  Support for environments that do not have SQL mode, such as OceanBase (Thanks to hanzhongzi for the contribution)
* Improvement PR-617:  Remove $has_child_tables var (Thanks to Ricardo for the contribution)
```

File: Changelog [hunk 2]
```diff
@@ -1,5 +1,13 @@
...
+* Fixed bug PT-2101: pt-mongodb-query-digest doesn't work on standalone server
+* Fixed bug PT-813: LP #1371774: pt-query-digest inconsistent rank order
+* Fixed bug PT-981: LP #1008240: t/pt-query-digest/processlist.t occasionally fails
+* Fixed bug PT-2229: CVE reported in Percona Toolkit version 3.5.3 related to Logrus (Thanks to Kushal Haldar for reporting this issue)
```
Comment (svetasmirnova) [inline Changelog]:
```suggestion
* Fixed bug PT-2229: CVE reported in Percona Toolkit version 3.5.3 related to Logrus (Thanks to Kushal Haldar for reporting this issue and testing the fix)
```

File: docs/release_notes.rst [hunk 3]
```diff
@@ -1,6 +1,22 @@
...
+
+* :jirabug:`PT-2151`: Fix tests for pt-query-digest
+
+Bugs Fixed
+------------
+
+* :jirabug:`PT-2101`: pt-mongodb-query-digest doesn't work on standalone server
+* :jirabug:`PT-813`:  LP #1371774: pt-query-digest inconsistent rank order
+* :jirabug:`PT-981`: LP #1008240: t/pt-query-digest/processlist.t occasionally fails
+* :jirabug:`PT-2229`: CVE reported in Percona Toolkit version 3.5.3 related to Logrus (Thanks to Kushal Haldar for reporting this issue)
```
Comment (svetasmirnova) [inline docs/release_notes.rst]:
```suggestion
* :jirabug:`PT-2229`: CVE reported in Percona Toolkit version 3.5.3 related to Logrus (Thanks to Kushal Haldar for reporting this issue and testing the fix)
```

File: docs/release_notes.rst [hunk 4]
```diff
@@ -1,6 +1,22 @@
...
+------------------------------------------------------------
+
+* :jirabug:`PT-2151`: Fix tests for pt-query-digest
+
+Bugs Fixed
+------------
+
+* :jirabug:`PT-2101`: pt-mongodb-query-digest doesn't work on standalone server
+* :jirabug:`PT-813`:  LP #1371774: pt-query-digest inconsistent rank order
+* :jirabug:`PT-981`: LP #1008240: t/pt-query-digest/processlist.t occasionally fails
```
Comment (svetasmirnova) [inline docs/release_notes.rst:17]:
```suggestion
* :jirabug:`PT-981`:  LP #1008240: t/pt-query-digest/processlist.t occasionally fails
```

**Review by svetasmirnova — CHANGES_REQUESTED** [review]
Please review the changes I suggested.

File: Changelog (hunk 1 above)
Comment (alina-derkach-oaza) [reply Changelog:5]: done

File: docs/release_notes.rst (hunk 3 above)
Comment (alina-derkach-oaza) [reply docs/release_notes.rst]: done

File: docs/release_notes.rst (hunk 4 above)
Comment (alina-derkach-oaza) [reply docs/release_notes.rst:17]: done

File: Changelog (hunk 2 above)
Comment (alina-derkach-oaza) [reply Changelog]: done

**Review by svetasmirnova — APPROVED** [review]

**Pull request merged**